docs/docs-content/release-notes/known-issues.md

```diff
@@ -16,6 +16,7 @@ The following table lists all known issues that are currently active and affecti
 | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------ | ---------------------------- |
+| The Certificate Authority (CA) certificate for Mutating Webhook Handler (MWH) expires after 90 days and does not get automaticallly renewed, which affects cluster health.                                                                                                                                                                                                                                                                                                                                                                                               | Access your clister [via kubectl](../clusters/cluster-management/palette-webctl.md) and issue the command `kubectl delete mutatingwebhookconfiguration palette-mutating-webhook-configuration` to manually delete the MWH and it will get recreated with a new certificate.                                                                                                                                                                                                                                                                                                                                     | Nov 4, 2024        | Edge                         |
```

Comment:
```suggestion
| The Certificate Authority (CA) certificate for Mutating Webhook Handler (MWH) expires after 90 days and does not get automatically renewed, which affects cluster health.                                                                                                                                                                                                                                                                                                                                                                                               | Access your cluster [via kubectl](../clusters/cluster-management/palette-webctl.md) and issue the command `kubectl delete mutatingwebhookconfiguration palette-mutating-webhook-configuration` to delete the MWH manually. It will then be recreated with a new certificate.                                                                                                                                                                                                                                                                                                                                     | Nov 4, 2024        | Edge                         |
```